ethpm/package.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, manifest: Dict[str, Any], w3: Web3) -> None:
         self.w3 = w3
         self.w3.eth.defaultContractFactory = LinkableContract
         self.manifest = manifest
+        self._uri = uri
 
     def set_default_w3(self, w3: Web3) -> "Package":
```

Comment:
@pipermerriam What do you think about something like `change_w3` or `update_w3` - since it no longer "sets" the w3 attr on a `Package` instance, but rather returns a new `Package` instance created with the given w3

Reply:
I could go with either with a *minor* inclination towards `update_w3`